core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation.directive.ts

```diff
@@ -56,8 +56,14 @@ angular.module('oppia').directive('editorNavigation', [
             postTutorialHelpPopoverIsShown: false
           };
           $scope.isLargeScreen = (WindowDimensionsService.getWidth() >= 1024);
-          $scope.isImprovementsTabEnabled =
-            ExplorationFeaturesService.isImprovementsTabEnabled;
+          $scope.isImprovementsTabEnabled = function() {
+            return ExplorationFeaturesService.isInitialized() &&
+              ExplorationFeaturesService.isImprovementsTabEnabled();
```

Comment:
The default value of `settings.isImprovementsTabEnabled` is `false`, so do we need to check `isInitialized`?

Reply:
Ah yes, we don't want to show either of the improvements/feedback tabs/buttons until we know for sure whether or not the improvement tab is enabled. Since the tabs/buttons get rendered before the service gets initialized, however, I perform this check to keep them both hidden until we know for sure.

Reply:
I mean, can we have the `isInitialized` inside the `isImprovementsTabEnabled` as both are coming from one service. In general, I was thinking to have one function from a service to confirm `isImprovementsTabEnabled`.

I'm fine with this as well, we can leave it as-is if you think it's fine this way! :)

Reply:
Ah I see, I think this is fine since it's only temporary (once we're comfortable with the Improvements Tab, we'll remove the feedback tab entirely and no longer have a need for these).